.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
Please do not include your editor settings.

Reply:
I think this project need a `.editorconfig`. If not include it, how to keep code style consistent?